scripts/build.sh

```diff
@@ -249,9 +249,7 @@ case "$product-$method-$platform" in
         "${xcb_flags[@]}" \
         build \
         test
-  ;;
```

Comment:
InAppMessagingDisplay used to build for iPad. `InAppMessaging-xcodebuild-iOS` above probably needs to become `InAppMessaging-xcodebuild-*`?